### PR TITLE
Update Changelog for 2.0.0.

### PR DESCRIPTION
**Pull request by keefertaylor**
## High Level Overview of Change

Cut 2.0.0 which has a few breaking changes and support for rippled protocol buffers.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### rippled support
+
+This release adds support for the new rippled protocol buffers and adds browser compatibility support.
+
+New Protocol Buffers from rippled
+rippled implemented protocol buffer support natively, which uses a new and incompatible set of protocol buffers. The implementation in rippled was completed in: ripple/rippled#3159. Xpring4j is now capable of using these protocol buffers. This change is opt-in and non-breaking.
+
+To switch to these protocol buffers, pass true to the useNewProtocolBuffer parameter in XpringClient's constructor. The default for this field is false. The remote rippled node must have gRPC enabled.
+
+### Additional fixes
```
Comment (hbergren) [inline CHANGELOG.md]:
```suggestion
### Additional Fixes
```
?

Not sure about the casing. `### rippled support` is all lowercase, but `### Breaking Changes` below is capitalized.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,7 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+This release contains some breaking changes that fix and productionize this library. In particular, users will now have to provide a URL for a remote gRPC node when using `XpringClient`. Clients can use `grpc.xpring.tech:80` if they weish to use Xpring's hosted TestNet service.
```
Comment (hbergren) [inline CHANGELOG.md]:
```suggestion
This release contains some breaking changes that fix and productionize this library. In particular, users will now have to provide a URL for a remote gRPC node when using `XpringClient`. Clients can use `grpc.xpring.tech:80` if they wish to use Xpring's hosted TestNet service. 
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,7 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+To switch to these protocol buffers, pass true to the useNewProtocolBuffer parameter in XpringClient's constructor. The default for this field is false. The remote rippled node must have gRPC enabled.
+
+### Additional fixes
+
+This release contains some breaking changes that fix and productionize this library. In particular, users will now have to provide a URL for a remote gRPC node when using `XpringClient`. Clients can use `grpc.xpring.tech:80` if they weish to use Xpring's hosted TestNet service. 
+
+Additionally, exceptions were mistakenly named `XpringKitException`, which was a reference to the Swift variant of this library. This is a simple rename, no functionality is added, changed or removed.
+
+### Breaking Changes
```
Comment (hbergren) [inline CHANGELOG.md]: Should this be `####` (H4) instead of H3? This seems like it falls under what you talk about in `### Additional Fixes`

**Comment by keefertaylor** [comment]
I am sorry this languished. 

I updated this changelog to also point out changes made last week with regard to the rippled protocol buffers. PTAL.

**Pull request merged**